optuna/importance/_base.py

```diff
@@ -44,13 +50,8 @@ def evaluate(
             target:
                 A function to specify the value to evaluate importances.
                 If it is :obj:`None` and ``study`` is being used for single-objective optimization,
-                the objective values are used. Can also be used for other trial attributes, such as
-                the duration, like ``target=lambda t: t.duration.total_seconds()``.
-
-                .. note::
-                    Specify this argument if ``study`` is being used for multi-objective
-                    optimization. For example, to get the hyperparameter importance of the first
-                    objective, use ``target=lambda t: t.values[0]`` for the target parameter.
+                the objective values are used. ``target`` must be specified if ``study`` is being
+                used for multi-objective optimization.
```

Comment:
if we delete the existing explanation of `target`, I think that we need to clarify that `target` returns a single float value.

Reply:
Sorry, I forgot to revert this part.